packages/components/buttons/primary-button/src/primary-button.styles.ts

```diff
@@ -45,7 +45,7 @@ const getButtonStyles = (
         &,
         &:active,
         &:hover {
-          background-color: ${designTokens.backgroundColorForButtonWhenDisabled};
```

Comment:
This token was used here and also in the `SecondaryButton` styles so perhaps it would make sense to keep it. 
It seems like a token all buttons may share.

What do you think?

Reply:
That makes sense, I left most of the button tokens as they seemed commonly shared. I guess I missed this one.